projects/packages/publicize/changelog/update-publicize-calypso-url

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I think this is probably right to be a minor change, as it's updating the links in the clasic editor, but as that means we need to update the version in Jetpack I wonder if we should separate the package change from the Social plugin change. 

Maybe we should get Crew's opinion on that?

Reply:
@jeherve, This PR has a change wrt redirect URLs for calypso. But, to tweak it, we had to change the publicize package. Should this PR be broken down into 2 different PR's - one for the Publicize package and the other for the plugin's file changes?

Reply:
Since this PR is already quite big, it may be worth doing a follow-up for the plugin's file changes?

Reply:
Thank @jeherve, I will do that.